recipes/balrog/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set version = "0.2.17" %}
+{% set sha256 = "ec1134f2e02ee44dfec04c8f37867b81747ee75db0b9d89f2005f53f8e6f0c7d" %}
+
+package:
+  name: balrog
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True  # [py!=37]
+
+source:
+  url: https://github.com/Markusjsommer/BalrogCPP/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - zlib
+    - pytorch-cpu >=1.5.0
+    - python
+  run:
+    - zlib
+    - pytorch-cpu >=1.5.0
+    - python
+
+test:
+  commands:
+    - balrog --help
+
+about:
+  home: https://github.com/Markusjsommer/BalrogCPP
+  license: MIT
```

@Markusjsommer, could you include a license file upstream which we could then add as a reference here with, e.g.,
```suggestion
  license: MIT
  license_file: LICENSE
```

---

Added!

---

Perfect, thanks!